frontend/cypress/integration/common/wizard_istio_config.ts

```diff
@@ -19,6 +19,20 @@ When('viewing the detail for {string}', (object: string) => {
   cy.get('a').contains(object).click();
 });
 
+When ('k8sgateway {string} is not created',(name:string) =>{
```

Comment:
please adjust this Gherkin and step definition to something a bit more comprehensible

`
user deletes k8sgateway named {string} and resource no longer available
`